gfx_scene.cpp

```diff
@@ -1268,34 +1268,49 @@ class GfxSceneInternal
             lights[&gltf_light] = light_ref;
         }
         std::map<cgltf_image const *, GfxConstRef<GfxImage>> images;
-        for(size_t i = 0; i < gltf_model->textures_count; ++i)
-        {
-            cgltf_texture const &gltf_texture = gltf_model->textures[i];
-            uint32_t texture_dds_index = std::numeric_limits<uint32_t>::max();
-            for(uint64_t e = 0; e < gltf_texture.extensions_count; ++e)
-            {
-                if(strcmp(gltf_texture.extensions[e].name, "MSFT_texture_dds") == 0)
-                {
-                    std::string json = gltf_texture.extensions[e].data;
-                    json.erase(remove_if(json.begin(), json.end(),
-                                   [](char c) { return (c == '\r' || c == '\t' || c == ' ' || c == '\n'); }),
+        std::map<cgltf_texture const *, GfxConstRef<GfxImage>> textures;
```

Comment:
Added an additional texture lookup for matching gltf textures with loaded gfx images. This is used as multiple gltf textures may reference the same gltf image. looking up the correct gltf image requires checking for presence of basisu or dds textures so for simplicity and speed we cache the texture to gfx image conversion.